karma.config.js

```diff
@@ -81,7 +81,7 @@ module.exports = function(karma) {
                 ].concat(coverage ? [
                   [ 'istanbul', {
                     include: [
-                      'lib/**'
+                      'src/**'
```

Comment:
Shouldn't this config and https://github.com/bpmn-io/dmn-js-properties-panel/blob/main/karma.config.js be the same ideally?

Reply:
Indeed, we could port this to the dmn project too.

Reply:
Oh, in DMN it just works. I guess there's no need to fix it then.

Reply:
Now this project will mirror the dmn-js-properties-panel settings. Thanks for the suggestion.